app/src/app/framework/category/category.component.html

```diff
@@ -1,24 +1,23 @@
-<h1>Category
```

Comment:
What was changed here?

Reply:
Here the div tag was inside the H1 tag.
This meant that the input field was not as long as the sidebar width.

Reply:
Ahh ok, I see - missed that.